app/assets/stylesheets/govuk_publishing_components/components/_button.scss

```diff
@@ -1,20 +1,18 @@
-@import "design-patterns/buttons";
-@import "mixins/media-down";
-@import "mixins/margins";
+// This component relies on styles from GOV.UK Frontend
 
-%gem-c-button,
-.gem-c-button {
-  @include button;
+// Specify the functions used to resolve assets paths in SCSS
+$govuk-image-url-function: "image-url";
+$govuk-font-url-function: "font-url";
 
-  @media(max-width: 425px) {
-    box-sizing: border-box;
-    width: 100%;
-    text-align: center;
-  }
+@import "../../../../node_modules/govuk-frontend/components/button/button";
+
+@import "mixins/margins";
 
-  &:focus {
-    outline: 3px solid $focus-colour;
-  }
+// Because govuk-frontend adds a responsive bottom margin by default for each component
+// we reset it to zero so we can set it separately using `gem-c-button--bottom-margin`
+// If we decide to use responsive margins consistently across components we can remove this
+.gem-c-button {
+  margin-bottom: 0;
```

Comment:
Could this instead be replaced with `.govuk-!-margin-bottom-0` in the template?

Reply:
yes, I can use the overwrite classes. good point.

Reply:
ah, I need to import the overwrite helpers for that, not sure if it's worth doing that for each component. Ideally we would settle for consistent spacing across component and we will remove these resets.

Reply:
Fair, not blocking, you do you. 👍